fe/fe-core/src/main/java/org/apache/doris/analysis/IndexDef.java

```diff
@@ -66,6 +93,19 @@ public void analyze() throws AnalysisException {
             if (columns.size() != distinct.size()) {
                 throw new AnalysisException("columns of index has duplicated.");
             }
+            if (arguments != null && !arguments.isEmpty()) {
+                throw new AnalysisException("bimap index do not need arguments.");
+            }
+        } else if (indexType == IndexType.NGRAM_BF) {
+            if (columns == null || columns.size() != 1) {
```

Comment:
common check for indexName can be moved to the outer scope.

Reply:
get